### ISSUE TITLE
Bazel locking fails on pdm 2.11.0 generated lock

### ISSUE THREAD

**Issue by Meriem-s** [jvolkman/rules_pycross#70]
I am getting the following error when attempting to generate a Bazel lock file from a `pdm.lock `  that was generated using pdm version 2.11.0 (or even higher) where `lock_version = "4.4.1"`

```
 File "/home/msaid/.cache/bazel/_bazel_msaid/7d8a2962841d975abf88710bbc9479a4/sandbox/linux-sandbox/1/execroot/cwbng/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.runfiles/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.py", line 279, in <module>
    app.run(main, flags_parser=parse_flags)
  File "/home/msaid/.cache/bazel/_bazel_msaid/7d8a2962841d975abf88710bbc9479a4/sandbox/linux-sandbox/1/execroot/cwbng/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.runfiles/rules_pycross_pypi_deps_absl_py/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/msaid/.cache/bazel/_bazel_msaid/7d8a2962841d975abf88710bbc9479a4/sandbox/linux-sandbox/1/execroot/cwbng/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.runfiles/rules_pycross_pypi_deps_absl_py/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/home/msaid/.cache/bazel/_bazel_msaid/7d8a2962841d975abf88710bbc9479a4/sandbox/linux-sandbox/1/execroot/cwbng/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.runfiles/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.py", line 211, in main
    lock_set = translate(
  File "/home/msaid/.cache/bazel/_bazel_msaid/7d8a2962841d975abf88710bbc9479a4/sandbox/linux-sandbox/1/execroot/cwbng/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.runfiles/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.py", line 190, in translate
    project = PDMProject(
  File "/home/msaid/.cache/bazel/_bazel_msaid/7d8a2962841d975abf88710bbc9479a4/sandbox/linux-sandbox/1/execroot/cwbng/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.runfiles/jvolkman_rules_pycross/pycross/private/tools/pdm_translator.py", line 43, in __init__
    raise LockfileIncompatibleException(
__main__.LockfileIncompatibleException: Lock file is not compatible with PDM version used by pycross.
Target //application/python/third_party:update_lock_file failed to build

```
Seems like this newer version of pdm (2.11.0) introduced an [update](https://github.com/pdm-project/pdm/pull/2421) where the `lock_version` field is changed from 4.4 to 4.4.1 and apparently this is not supported in [here](https://github.com/jvolkman/rules_pycross/blob/main/pycross/private/tools/pdm_translator.py#L43).

**Comment by jvolkman** [comment]
Hi,
It looks like this should be fixed by #56 which is included in the next release, which I'm hopeful will be out today.

You can find the draft [here](https://github.com/jvolkman/rules_pycross/releases/tag/v0.5.0) will be note that it's a draft and may change.

**Comment by jvolkman** [comment]
Feel free to reopen if this is still an issue for you with the latest release.
